server.js

```diff
@@ -73,16 +71,17 @@ app.post('/up', function(req, res) {
                 });
             }
         } catch (err) {
-            console.log("Error on finish:", err);
+            console.error("Error on finish:", err);
             res.send("Internal Server Error");
         }
     });
 
     return req.pipe(busboy);
-});
+};
+
 
-app.get('/del', function(req, res) {
-    var d = fields[blah];
+function handledelete(req, res) {
+    var config = req.app.locals.config
```

Comment:
I don't think so, it makes more sense for the app to just own it's own config rather than binding event handlers to own the config.